docs/concepts/executor/serve.md

```diff
@@ -157,6 +155,33 @@ The above example deploys the `DummyHubExecutor` from Executor Hub into your Kub
 The Executor you use needs to be already containerized and stored in a registry accessible from your Kubernetes cluster. We recommend Executor Hub for this.
 ````
 
+Once the Executor is deployed, you can expose a service:
+```bash
+kubectl expose deployment executor --name=executor-exposed --type LoadBalancer --port 80 --target-port 8080 -n my-namespace
+sleep 60 # wait until the external ip is configured
+```
+
+Let's export the external IP address created and use it to send requests to the Executor. 
+```bash
+export EXTERNAL_IP=`kubectl get service executor-exposed -n my-namespace -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+```
+To send requests using {meth}`~jina.Client`, make sure to set `stream=False`:
```

Comment:
If a Gateway is added by default then there is no need to set `stream=False`.

Reply:
My misunderstanding again. The `include_gateway` is used only in the overload method in the Flow.